.devcontainer/devcontainer.json

```diff
@@ -1,26 +1,32 @@
 {
   "hostRequirements": {
     "cpus": 4,
-    "memory": "8gb",
+    "memory": "7gb",
     "storage": "32gb"
   },
   "name": "Renovate",
   "dockerFile": "Dockerfile",
-  "settings": {
-    "terminal.integrated.profiles.linux": { "bash": { "path": "/bin/bash" } },
-    "terminal.integrated.defaultProfile.linux": "bash"
+  "context": "..",
+  "customizations": {
+    "vscode": {
+      "settings": {
+        "terminal.integrated.profiles.linux": {
+          "bash": { "path": "/bin/bash" }
+        },
+        "terminal.integrated.defaultProfile.linux": "bash"
+      },
+      "extensions": [
+        "dbaeumer.vscode-eslint",
+        "esbenp.prettier-vscode",
+        "orta.vscode-jest",
+        "EditorConfig.editorconfig"
+      ]
+    }
   },
-  "extensions": [
-    "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode",
-    "orta.vscode-jest",
-    "EditorConfig.editorconfig"
-  ],
   "runArgs": [
     "--cap-add=SYS_PTRACE",
     "--security-opt",
     "seccomp=unconfined",
     "--privileged"
-  ],
-  "postCreateCommand": "yarn install"
```

Comment:
revert this

Reply:
why?

Reply:
because this is executed by vs code when opening the folder inside the devcontainer.